src/screens/Notification.js

```diff
@@ -0,0 +1,314 @@
+import React from 'react';
+import { Button } from 'grommet';
+import Page from '../components/Page';
+import Item from './Components/Item';
+import { GenericColor } from '../utils/genericThemeExamples';
+import {
+  ComponentDoc,
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+  ThemeDoc,
+} from '../components/Doc';
+
+export default () => (
+  <Page>
+    <ComponentDoc
+      name="Notification"
+      availableAt={[
+        {
+          url: 'https://storybook.grommet.io',
+          badge: 'https://cdn-images-1.medium.com/',
+          label: 'Storybook',
+        },
+        {
+          url: 'https://codesandbox.io/s/github',
+          badge: 'https://codesandbox.io/',
+          label: 'CodeSandbox',
+        },
+      ]}
+      description="A Notification"
+      code="// <Notification />"
```

Comment:
Right now the code example is causing a syntax error. Maybe change it to {`// <Notification message="..." /> `} and then add more and uncomment once Notification has been added to Grommet.